tests/Microsoft.Identity.Test.LabInfrastructure/KeyVaultSecretsProvider.cs

```diff
@@ -144,19 +146,37 @@ private async Task<string> AuthenticationCallbackAsync(string authority, string
                             "Test setup error - cannot find a certificate in the My store for KeyVault. This is available for Microsoft employees only.");
                     }
 
-                    app = ConfidentialClientApplicationBuilder.Create(_config.ClientId)
+                    confidentialApp = ConfidentialClientApplicationBuilder.Create(_config.ClientId)
```

Comment:
Might be clearer to not use _config.ClientId at all, just use the constants directly.